src/ViewHandler/SecuredViewHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace PhpList\RestBundle\ViewHandler;
+
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\View\ViewHandler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This class is used to add headers to the default response.
+ *
+ * @author Xheni Myrtaj <xheni@phplist.com>
+ */
+class SecuredViewHandler
+{
+    /**
+     * @param ViewHandler $viewHandler
+     * @param View $view
+     * @param Request $request
+     * @param string $format
+     *
+     * @return Response
+     */
+    public function createResponse(ViewHandler $handler, View $view, Request $request, string $format)
```

Comment:
Would it work to have `Response` as a return type declaration?

Reply:
Yes, it works.